tests/test_serialization.py

```diff
@@ -0,0 +1,39 @@
+import fv3config
+import io
+import datetime
+import pytest
+from toolz import assoc, dissoc
```

Comment:
would prefer to not add this dep.

Reply:
https://github.com/VulcanClimateModeling/fv3config/pull/131#issuecomment-791649574

Reply:
removed